test/e2e/cmd/rabbitproducer/main.go

```diff
@@ -77,6 +77,7 @@ func main() {
 			false,  // mandatory
 			false,  // immediate
 			amqp.Publishing{
+				Headers:     amqp.Table{"content-type": "text/plain"},
```

Comment:
So this means that for every msg in the rabbitmq, we require content-type? Is that ok? Just curious if that would then mean that folks would have to change the messages they produce? Or put another way, is there a way to assume that if there's none, we would treat missing as text/plain?
@n3wscott since he also has looked at Rabbit.

Reply:
Yup i think i need a way to get the content type property, but i cant find how to do it in the wabbit delivery interface. Need a little help with that